app/containers/Applications/ApplicationCommentsByForm.tsx

```diff
@@ -35,12 +37,32 @@ export const ApplicationCommentsByForm: React.FunctionComponent<Props> = props =
     console.log(response);
   };
 
+  const deleteComment = async () => {
+    const {environment} = props.relay;
+    const variables = {
+      input: {
+        id: reviewComment.id,
+        reviewCommentPatch: {
+          deletedAt: moment
+            .tz('America/Vancouver')
+            .format('YYYY-MM-DDTHH:mm:ss')
+        }
+      },
+      applicationId: reviewComment.applicationId,
```

Comment:
```suggestion
      applicationId: reviewComment.applicationByApplicationId.id,
```
This is the relay id, reviewComment.applicationId is the numbered (row) id

Reply:
Updated